src/Microsoft.AspNetCore.Blazor/Components/IHandleEvent.cs

```diff
@@ -11,8 +13,8 @@ public interface IHandleEvent
         /// <summary>
         /// Notifies the component that one of its event handlers has been triggered.
         /// </summary>
-        /// <param name="binding">The event binding.</param>
-        /// <param name="args">Arguments for the event handler.</param>
-        void HandleEvent(EventHandlerInvoker binding, UIEventArgs args);
+        /// <param name="invoker">The event invoker.</param>
+        /// <param name="eventArgs">Arguments for the event handler.</param>
+        Task HandleEvent<TArg>(EventHandlerInvoker<TArg> invoker, TArg eventArgs);
```

Comment:
HandleEventAsync? - have we been using async as a suffix on lifecycle methods?

Reply:
Good point - will change this.